torch/onnx/symbolic_opset12.py

```diff
@@ -47,16 +47,26 @@ def outer(g, input, other):
     return einsum_helper(g, "i,j->ij", [input, other])
 
 
-@symbolic_helper.parse_args("v", "f", "i")
-def dropout(g, input, p, train):
+def _dropout_shared(g, input, p, train):
```

Comment:
```suggestion
def _dropout_shared(g: _C.Graph, input: _C.Value, p: float, train: bool) -> Tuple[_C.Value, Optional[_C.Value]]:
```

assuming mask is a Value?

Reply:
As I mentioned in another comment, no exporter does type annotation because it will trigger an exception in CIs. Someone needs to fix the root cause. You can reproduce it by running
```
lintrunner -a master
```
and it will error out with
```
  Error (MYPY) [attr-defined]
    "Graph" has no attribute "op"
```

Reply:
Let’s omit graph and annotate the rest for now?

Reply:
Ok. This is the new signature:
```python
def _dropout_returns_masked_input_and_mask(
    g, input: torch._C.Value, p: float, train: bool
):
```

Reply:
_C.Value is good, for consistency